launcher/cli/logging.go

```diff
@@ -100,6 +104,58 @@ func setupLogger() {
 
 	// Hijack standard Golang `log` and redirect it to our common logger
 	zap.RedirectStdLogAt(commonLogger, zap.DebugLevel)
+
+	if listenAddr != "" {
+		go func() {
+			userLog.Debug("starting atomic level switcher on" + listenAddr)
+			if err := http.ListenAndServe(listenAddr, http.HandlerFunc(handleHTTPLogChange)); err != nil {
+				userLog.Warn("failed listening on :1065 to switch log level:", zap.Error(err))
```

Comment:
The `1065` might not be quite true depending on the argument passed.
```suggestion
				userLog.Warn("failed starting atomic level switcher http server", zap.Error(err))
```